catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/types/CoreAttributes.java

```diff
@@ -84,7 +84,7 @@
         descriptors.add(new AttributeDescriptorImpl(METADATA,
                 true /* indexed */,
                 true /* stored */,
-                true /* tokenized */,
+                false /* tokenized */,
```

Comment:
Shouldn't `BasicTypes` be deprecated in favor of `Core`/`CoreAttributes` anyways?